CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The public API of this library consists of the functions declared in file
 [h3api.h.in](./src/h3lib/include/h3api.h.in).
 
 ## [Unreleased]
+### Added
+- `h3` binary for shell scripts ready for use (#931)
```

Comment:
```suggestion
- `h3` binary for shell scripts ready for use (#818, #826, #846, #923, #924, #931)
```
You could mention more of your PRs that have gone into the CLI if you like

Reply:
Yes, I wasn't sure how far back I should go on that front, though.